config.c

```diff
@@ -332,7 +336,8 @@ static int pv_config_save_creds_to_file(struct pantavisor_config *config, char *
 	int fd;
 	char tmp_path[PATH_MAX];
 
-	sprintf(tmp_path, "%s-XXXXXX", path);
+	if (!sprintf_path(tmp_path, "%s-XXXXXX", path))
+		pv_log(WARN, "tmp_path truncated to %s", tmp_path);
```

Comment:
maybe we could put this in a macro somewhere in utils to avoid repeating it through the code?

Reply:
I thought about this. My probelm with a macro is that we cannot check whether pv_log is ready to use or not. This is the next thing I want to work on: making pv_log print to stdout until the pantavisor logger is ready so that we create the macro.

Reply:
It's OK then :)